testing/frr/frr.pre-install

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+for g in frr frrvty; do
+	addgroup -S $g 2>/dev/null
+done
+
+adduser -S -D -H -h /var/run/frr -s /sbin/nologin -G frr -g frr frr 2>/dev/null
```

Comment:
I wonder if it would make more sense to use `/var/empty` as home directory. `/var/run` is a symlink to `/run` which is mounted as tmpfs, which means that the subdirectory `/run/frr` will be removed on reboot.

Reply:
Right, these are socket and vty files, they should be on tmpfs, nothing is preserved across reboots there...